test/common/api_utils.py

```diff
@@ -0,0 +1,170 @@
+from on_http_api2_0 import ApiApi as Api
+from config.api2_0_config import config
+from json import loads
+import sys
+import os
+
+
+def api_node_select_argsList(nodeType='compute', validate_obm=False):
+    """
+    This routine produces a list of node Id's that satisfy the requested parameters. 
+    Nodes may be filtered by Node Id,  SKU name, OBM MAC address which are defined
+    in the system environment.  The node type and OBM setting may be specified.
+    :param  nodeType: Node type (i.e. compute) (default= 'compute')
+    :param  validate_obm: should the node have OBM settings (default=False)
+    return: A list with node IDs that match node type, SKU, and possible valid OBM settings.
+    """
+    return api_node_select(config.api_client,
+                           nodeId=os.getenv("NODEID", "None"),
+                           skuName=os.getenv("SKU", "all"),
+                           obmmac=os.getenv("OBMMAC", "all"),
+                           nodeType=nodeType,
+                           validate_obm=validate_obm,
+                           verbosity=os.getenv("VERBOSITY", "0")
+                           )
+
+
+
+def api_node_select(client,
+                    nodeId=None,
+                    skuName='all',
+                    obmmac='all',
+                    nodeType='compute',
+                    validate_obm=False,
+                    verbosity=None
+                    ):
```

Comment:
Considering the flexibility if we need further filter criteria in future and the easy to use because developer hard to remember all these argument name and order, I suggest putting all these criterias into a dict, below is the sample code:

```python
def api_node_select(client, criteria, verbosity):
  skuName = criteria.get('skuName', 'all')
  nodeType = criteria.get('nodeType', 'compute')
  ...
```